cmd/api/protos/add-mood.go

```diff
@@ -0,0 +1,21 @@
+package protos
```

Comment:
Dashes in file names ? I can't find the doc, but I remember something in the lines of "use underscores".
Or simply `addmood.go` ?